openstack/nova/templates/etc/_nova.conf.tpl

```diff
@@ -74,13 +74,17 @@ port = {{ .Values.consoles.shellinabox.portInternal }}
 base_url = https://{{include "nova_console_endpoint_host_public" .}}:{{ .Values.global.novaConsolePortPublic }}/shellinabox
 proxyclient_url = https://{{include "ironic_console_endpoint_host_public" .}}
 
+[mks]
```

Comment:
I would say, this probably should rendered dependent on the value set here:
https://github.com/sapcc/helm-charts/pull/500/files#diff-e5e918ce7b67ccfe478849a3bcc2dfb1R152

Reply:
I don't understand.

Reply:
In the values.yaml, you write under `consoles.mks` that setting the values will enable the the mks console.
I think, it would be sensible, that the conf file would follow that. Meaning, only to add the section, if the settings are there.

Reply:
ok